frontend/components/Filters/DoubleSelect/index.tsx

```diff
@@ -0,0 +1,177 @@
+                  <span className="absolute inset-y-0 right-0 flex items-center pr-2 pointer-events-none">
+                    <SelectorIcon
+                      className="h-5 w-5 text-gray-400"
+                      aria-hidden="true"
+                    />
+                  </span>
+                </Listbox.Button>
+
+                <Transition
+                  show={open}
+                  as={Fragment}
+                  leave="transition ease-in duration-100"
+                  leaveFrom="opacity-100"
+                  leaveTo="opacity-0"
+                >
+                  <Listbox.Options className="absolute z-10 mt-1 w-full bg-white shadow-lg max-h-60 rounded-md py-1 text-base ring-1 ring-black ring-opacity-5 overflow-auto focus:outline-none sm:text-sm">
+                    {selectOptions.map((person: Option) => (
+                      <Listbox.Option
+                        key={person.id}
+                        className={({ active }) =>
+                          classNames(
+                            active
+                              ? "text-white bg-indigo-600"
+                              : "text-gray-900",
+                            "cursor-default select-none relative py-2 pl-3 pr-9"
+                          )
+                        }
+                        value={person}
+                      >
+                        {({ selected, active }) => (
+                          <>
+                            <span
+                              className={classNames(
+                                selected ? "font-semibold" : "font-normal",
+                                "block truncate"
+                              )}
+                            >
+                              {person.name}
+                            </span>
+
+                            {selected ? (
+                              <span
+                                className={classNames(
+                                  active ? "text-white" : "text-indigo-600",
+                                  "absolute inset-y-0 right-0 flex items-center pr-4"
+                                )}
+                              >
+                                <CheckIcon
+                                  className="h-5 w-5"
+                                  aria-hidden="true"
+                                />
+                              </span>
+                            ) : null}
+                          </>
+                        )}
+                      </Listbox.Option>
+                    ))}
+                  </Listbox.Options>
+                </Transition>
+              </div>
+            </>
+          )}
+        </Listbox>
+        <span className={"mt-3 px-2"}>do</span>
+        <Listbox value={selectedSecond} onChange={setSelectedSecond}>
+          {({ open }) => (
+            <>
+              <div className="mt-1 relative min-w-doubleFiltersSize">
```

Comment:
Since this node's parent is a flex, can't this just fill its parent? I do not understand why you needed to set min-width property on it in order to achieve full width.